rules/codelingo/effective-go/defer-close-file/codelingo.yaml

```
@@ -4,10 +4,29 @@ tenets:
       codelingo/docs:
         title: Defer Close File
         body: |
-          Deferring a call to a function such as Close has two advantages. First, it guarantees that you will never forget to close the file, a mistake that's easy to make if you later edit the function to add a new return path. Second, it means that the close sits near the open, which is much clearer than placing it at the end of the function.
-          TODO if a function returns the open file, follow it and check if it is closed
+          Deferring a call to a function such as Close has it's advantages. Firstly, it guarantees that 
+          you will never forget to close the file, a common mistake if you later edit the function 
+          to add a new return path. Secondly, it means that the Close sits near the open of the function, which is clearer 
+          than placing it at the end.
+          TODO if a function returns the open file, follow it and check if it is closed.
+
+          ## Bad code
+          ```golang
+          // Contents returns the file's contents as a string but does not defer the function.
```

```suggestion
          // Contents returns the file's contents as a string.
```

I reckon just use this comment for bot examples to make it more realistic.